homeassistant/components/knx/__init__.py

```diff
@@ -371,11 +358,11 @@ def init_xknx(self) -> None:
         """Initialize XKNX object."""
         self.xknx = XKNX(
             own_address=self.config[CONF_KNX_INDIVIDUAL_ADDRESS],
-            rate_limit=self.config[ConnectionSchema.CONF_KNX_RATE_LIMIT],
-            multicast_group=self.config[ConnectionSchema.CONF_KNX_MCAST_GRP],
-            multicast_port=self.config[ConnectionSchema.CONF_KNX_MCAST_PORT],
+            rate_limit=self.config[CONF_KNX_RATE_LIMIT],
+            multicast_group=self.config[CONF_KNX_MCAST_GRP],
+            multicast_port=self.config[CONF_KNX_MCAST_PORT],
```

Comment:
We should probably use these from the ConfigEntry directly instead of merging them back to the yaml Config.

Reply:
Yes, I have dropped the merging part and we solely can use config entry data now!